catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/CatalogCommands.java

```diff
@@ -66,14 +66,13 @@
 
   // DDF-535: remove "-provider" alias in DDF 3.0
   @Option(
-    name = "--provider",
```

Comment:
❓ Can the formatter be changed to only use a two-space indent for multi-line annotations? It'd be nice to reduce the files changed by what looks like an arbitrary style change.

Reply:
Won't changing the defaults make it drift from the various `fmt` IDE plugins?

Reply:
Verified that the latest intellij plugin will format the same as the maven plugin. So need to be in sync on the versions. https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml.